css/components/navbar.css

```diff
@@ -0,0 +1,81 @@
+@import "../utilities/variables.css";
+@import "../helpers/typography.css";
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+
+.navbar{
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: var(--size-9);
+}
+
+.navbar-brand {
+    padding-top: var(--size-4);
+    padding-bottom: var(--size-4);
+    margin-right: var(--size-9);
+    font-size: var(--size-10);
+    text-decoration: none;
+    white-space: nowrap;
+}
+
+.navbar-brand a{
+    color: var(--black);
+}
+
+.navbar-nav{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    list-style-type: none;
+}
+
+ .nav-item .nav-link{
+    display: inline-block;
+    padding: var(--size-6) var(--size-10);
+    color: var(--grey);
+}
+
+.hamburger{
+    background-color: transparent;
+    border: 0;
+    font-size: var(--size-10);
+    cursor: pointer;
+    display: none;
+}
+
+.hamburger:focus{
+    outline: none;
```

Comment:
This will make the hamburger unaccessible to the keyboard users.

Reply:
Then how to make the hamburger accessible to the keyboard users.